ls1intum/Apollon
# remove form from petri-net-place -> solves reloading on enter

### Motivation and Context



Fixes #144 

### Description

removing the form in the update dialog solves the issue. In html form on enter the form is submitted and the page reloaded

### Steps for Testing


1. create petri-net
2. create a petri net place
3. change capacity and press "Enter" on the keyboard 
4. Expected behavior: No reload occurs
